### DISCUSSION TITLE
Add azure-roles

### DISCUSSION THREAD

**Discussion by raix** [platformplatform/PlatformPlatform#639] [Ideas]
### Detailed description?

**Is your feature request related to a problem? Please describe.**
Adding / maintaining resource roles requires manual work looking up roles in azure.

**Describe the solution you'd like**
Adding the json file from [azure-roles](https://github.com/microsoft/azure-roles) allows for an easy way to keep roles updated and working with roles.

**Describe alternatives you've considered**
-

**Additional context**
-

### Feature Type

Infrastructure

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by tjementum** [comment]
I'm not quite sure what the idea is here.

For now, I converted this to a discussion. I'm trying to keep the issues clean so they do not contain a lot of items that are just ideas and will not be done in the short run.

**Reply by raix** [reply]
The https://github.com/microsoft/azure-roles enables autocomplete / "type safety" when using azure roles. Currently PP only use a few roles, but the second you add AI / ServiceBus etc. it becomes harder to navigate.
